scss/components/_legal-search.scss

```diff
@@ -45,14 +45,30 @@
   }
 }
 
-.legal-mur__archive {
-  font-size: u(1.3rem);
-  margin-top: u(0.8rem);
-
-  .legal-mur__archive-icon {
-    @include u-icon($file-box, $base, 1.5rem, 1.5rem, 100%);
-    display: inline-block;
-    margin-right: u(0.7rem);
-    vertical-align: top;
+.legal-mur {
+  .legal-mur__archive {
+    font-family: $sans-serif;
+    font-size: u(1.3rem);
+    margin-top: u(0.8rem);
+
+    .legal-mur__archive-icon {
+      @include u-icon($file-box, $base, 1.5rem, 1.5rem, 100%);
+      display: inline-block;
+      margin-right: u(0.7rem);
+      vertical-align: top;
+    }
+  }
+
+  .legal-mur__property {
+    margin-bottom: u(1rem);
+  }
+
+  .legal-mur__subject-list {
```

Comment:
Ah hmmm. That might be harder to change because it's used a lot. That approach could work, or we could have that stay the same and a new `.list--dense` or something? Or just using your approach here works too. I don't have strong feelings.